doc/maintaining/cli.rst

```diff
@@ -0,0 +1,558 @@
+
+For example::
+
+ ckan -c /path/to/ckan.ini --plugin=ckan search-index rebuild 
+
+This default behaviour will clear the index and rebuild it with all datasets. If you want to rebuild it for only
+one dataset, you can provide a dataset name::
+
+    ckan -c /path/to/ckan.ini --plugin=ckan search-index rebuild test-dataset-name 
+
+Alternatively, you can use the `-o` or `--only-missing` option to only reindex datasets which are not
+already indexed::
+
+    ckan -c /path/to/ckan.ini --plugin=ckan search-index rebuild -o 
+
+If you don't want to rebuild the whole index, but just refresh it, use the `-r` or `--refresh` option. This
+won't clear the index before starting rebuilding it::
+
+    ckan -c /path/to/ckan.ini --plugin=ckan search-index rebuild -r 
+
+There is also an option available which works like the refresh option but tries to use all processes on the
+computer to reindex faster::
+
+    ckan -c /path/to/ckan.ini --plugin=ckan search-index rebuild_fast 
+
+There are other search related commands, mostly useful for debugging purposes::
+
+    search-index check                  - checks for datasets not indexed
+    search-index show DATASET_NAME      - shows index of a dataset
+    search-index clear [DATASET_NAME]   - clears the search index for the provided dataset or for the whole ckan instance
+
+
+seed: Create test data in the database
+==================================
+
+Examples::
+
+      ckan -c /path/to/ckan.ini seed 
+      
+      
+server: Start Development server
+==================================
+
+Usage::
+
+    server --host (-h)          - Set Host
+    server --port (-p)          - Set Port
+    server --reloader (-r)      - Use reloader
+    
+
+sysadmin: Give sysadmin rights
+==============================
+
+Gives sysadmin rights to a named user. This means the user can perform any action on any object.
+
+For example, to make a user called 'admin' into a sysadmin::
+
+ ckan -c /path/to/ckan.ini --plugin=ckan sysadmin add admin 
+
+
+tracking: Update tracking statistics
+====================================
+
+Usage::
+
+    tracking update [start_date]       - update tracking stats
+    tracking export FILE [start_date]  - export tracking stats to a csv file
+
+
+translation: Translation helper functions
+===================================
+
+Usage::
+
+    trans js        - generate the JavaScript translations
+    trans mangle    - mangle the zh_TW translations for testing
+
+.. note::
+
+    Since version 2.7 the JavaScript translation files are automatically
+    regenerated if necessary when CKAN is started. Hence you usually do not
+    need to run ``ckan trans js`` manually.
+
+
+.. _cli-user:
+
+user: Create and manage users
+=============================
+
+Lets you create, remove, list and manage users.
+
+For example, to create a new user called 'admin'::
+
+ ckan -c /path/to/ckan.ini --plugin=ckan user add admin 
```

Comment:
```suggestion
 ckan -c |ckan.ini| user add admin 
```

Reply:
Ah I see @smotornyuk previously mentioned all the `--plugin` parameters - maybe do a global search?